src/resources/Edit/Alteration/cooperative.ts

```diff
@@ -0,0 +1,19 @@
+import { CorpTypeCd, FilingCodes, NameRequestEntityTypes } from '@/enums'
+import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+import { ResourceIF } from '@/interfaces'
+
+export const CooperativeResource: ResourceIF = {
+  entityType: CorpTypeCd.COOP,
+  entityReference: 'Company',
+  contactLabel: 'Registered Office',
+  displayName: GetCorpFullDescription(CorpTypeCd.COOP),
+  nameRequestType: NameRequestEntityTypes.CP,
+  filingData: {
+    entityType: CorpTypeCd.COOP,
+    filingTypeCode: FilingCodes.ALTERATION,
+    priority: false
+  },
+  certifyClause: 'Note: It is an offence to make a false or misleading statement in respect ' +
+    'of a material fact in a record submitted to the Corporate Registry for filing. ' +
+    'See section 427 of the Business Corporations Act.'
```

Comment:
Double check the act here. Might be different for coops.

Reply:
haha i was thinking the same. However, we don't have Alterations available for Coops to date, so in the event we implement them this can be updated!

Reply:
How about dropping a "FUTURE" comment in here so we remember to update it?

Reply:
Will do, updated!